dist/scripts/record-benchmark-demo/record-benchmark-demo

```diff
@@ -0,0 +1,64 @@
+#! /usr/bin/env bash
+
+# CC0 1.0 Unvanquished Developers
+# https://creativecommons.org/publicdomain/zero/1.0/
+
+# Record a demo purposed for benchmark.
+#
+# Can be run this way:
+#   ./record-benchmark-demo
+# Or this way:
+#   ./record-benchmark-demo /path/to/custom/daemon -customOptions +customCommands
+#
+# This will output a file named benchmark.dm_86
+
+set -e
+set -u
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+
+if [ "$(uname -s)" != 'Linux' ]
+then
+	echo 'ERROR: This script expects to run on Linux for now.' >&2
+	exit 1
+fi
+
+TMPDIR="${TMPDIR-/tmp}"
+
+temp_home_path="$(mktemp -d "${TMPDIR}/unv-benchmark-home-XXXXXXXX")"
+config_dir="${temp_home_path}/config"
+demo_dir="${temp_home_path}/demos"
+
+home_path="${XDG_DATA_HOME:-${HOME}/.local/share}/unvanquished"
+lib_path="${home_path}/base"
+
+daemon_path="${1:-${lib_path}/daemon}"
+shift || true
+
+mkdir -p "${config_dir}"
+cp -a "${script_dir}/record-benchmark-demo.cfg" "${config_dir}"
+
+"${daemon_path}" \
+	-homepath "${temp_home_path}" \
+	-set name 'benchmark' \
+	-set common.shutdownOnDrop on \
+	-set common.framerate.max 0 \
+	-set common.framerate.maxMinimized 0 \
```

Comment:
Is there some reason to max out the frame rate instead of limiting it to 100 or something?

Reply:
Hmm, this is not deeply thought, I just wanted to be sure nothing was an obstacle for anything.

Also I was concerned for minimized/unfocused things because of:

- https://github.com/DaemonEngine/Daemon/issues/613

But I'm still trying to figure out things, there may be useless things. I even don't know if we need to set an high framerate. I have also another question: does it have an impact when running `daemon-tty`?